code/modules/projectiles/projectile/special.dm

```diff
@@ -11,6 +11,16 @@
 	empulse(A, pulse_range, pulse_range)
 	return 1
 
+
+/obj/item/projectile/ion/stun/on_impact(var/atom/A)
+	if(isipc(A))
+		var/mob/living/carbon/human/H = A
+		H.Weaken(5)
+		to_chat(H, "<span class='danger'>ERROR: detected low setting EMP, acutators experience temporary power loss. Attempting to restore power.</span>")
+	else
+		empulse(A, pulse_range - 1, pulse_range - 1)

```

Comment:
I dont like the `pulse_range -1`

Reduce the pulse_range in the projectile definition accordingly (to 0)

Reply:
Changed to emp_act instead